src/linker/Linker/TypeMapInfo.cs

```diff
@@ -166,10 +167,12 @@ void MapVirtualMethods (TypeDefinition type)
 				return;
 
 			foreach (MethodDefinition method in type.Methods) {
-				if (!method.IsVirtual)
+				// This could be a static interface method implementation that has a .override
+				if (!(method.IsStatic || method.IsVirtual))
```

Comment:
I see, we want to remember the overrides but not to base mapping.

I would still do something along the lines of this.

```csharp
if (!method.IsVirtual)
    continue;

if (!method.IsStatic)
    MapVirtualMethod (method);

if (method.HasOverrides)
    MapOverrides (method);
```

Maybe it's just me, but it took me a while to realize when the `(!(method.IsStatic || method.IsVirtual))` condition is **not taken** (since this is just a continue, I'm more interested in when this evaluates to false).